games-action/polymc/files/polymc-explicit_include_in_PollServer.patch

```diff
@@ -0,0 +1,26 @@
+From c2b3567cb18e81a5f82c95c2c323b19488d62ed8 Mon Sep 17 00:00:00 2001
+From: Thiago Donato Ferreira <flowlnlnln@gmail.com>
+Date: Sat, 10 Sep 2022 17:59:27 -0300
+Subject: [PATCH] fix: add explicit QDebug include in PollServer
+
+Signed-off-by: Thiago Donato Ferreira <flowlnlnln@gmail.com>
+---
+ libraries/katabasis/src/PollServer.cpp | 2 ++
+ 1 file changed, 2 insertions(+)
+
```

Comment:
We usually strip most of the patch's header to save space since most of the information in there is not relevant for us.

Reply:
Oh sorry, the quazip patches have it all so I just did the same :c